src/Patches.cs

```diff
@@ -128,7 +141,7 @@ private static void SaveInformation(bool saveToFile)
 					RecordData(line, "Scene Definition", saveToFile);
 					return;
 				case GrabberMode.LootTable:
-					GameObject gameObject1 = GameManager.GetPlayerManagerComponent()?.m_InteractiveObjectNearCrosshair;
+					GameObject? gameObject1 = GameManager.GetPlayerManagerComponent().GetInteractiveObjectUnderCrosshairs(pickupRange);
 
 					if (gameObject1 is null) return;
```

Comment:
Anywhere you see `anythingInheritingFromUnityEngineObject is null`, it needs changed to `== null`. They are not compatible with pattern matching and will create race conditions.

Similar issues:

- `is not null`
- `component?.gameObject`